package.json

```diff
@@ -23,8 +23,9 @@
         "prettier": "prettier --write \"./{src,test}/*.{js,ts,tsx}\"",
         "lint": "eslint . --ext .js,.ts,.tsx",
         "validate": "tsc --noEmit",
-        "test": "jest --watch",
-        "test:ci": "jest -i --coverage",
+        "jest": "node --expose-gc $(yarn bin jest)",
```

Comment:
This will not work on windows machines.
Any idea what we can do there?

Reply:
And why do we need it exactly?

Reply:
On the Finalization registry group test we use that to force GC so the cleanup will run.
I'v tried other ways without much success.
I can give it another try

Reply:
Maybe this could help? https://www.npmjs.com/package/expose-gc

Reply:
It doesn't work on windows because of the bash not because of `--expose-gc` (that works on windows).

`$(yarn bin jest)` expands to the jest location. I think since the path is already supposed to be set I would use `NODE_FLAGS` with `cross-env` (to set them up in Windows).

Alternatively, I think it's fine not to run this test on Windows.

Reply:
NODE_FLAGS dose not work with expose gc unfortunately,
Using that package seems working, i will push it soon